evap/staff/forms.py

```diff
@@ -361,6 +368,18 @@ def clean_email(self):
     def save(self, *args, **kw):
         super().save(*args, **kw)
         self.instance.course_set = list(self.instance.course_set.exclude(semester=Semester.active_semester())) + list(self.cleaned_data.get('courses_participating_in'))
+        
+        staff_group = Group.objects.get(name="Staff")
+        grade_user_group = Group.objects.get(name="Grade publisher")
+        if (self.cleaned_data.get('is_staff')):
+            staff_group.user_set.add(self.instance)
+        else:
+            staff_group.user_set.remove(self.instance)
+
+        if (self.cleaned_data.get('is_grade_user')):
+            grade_user_group.user_set.add(self.instance)
+        else:
+            grade_user_group.user_set.remove(self.instance)
```

Comment:
ifs without the brackets please :) and i'd prefer `self.instance.groups.add(group)` over `group.user_set...` it makes it clearer that we're modifying a user and not a group.